docs/content/getting-started.md

```diff
@@ -42,6 +42,7 @@ you should adjust to your own environment.
     !!! important
 
         In order to access applications in your guest clusters, the public zone must be routable.
+        Please skip this step if the public zone exists, otherwise it will affect the existing functions.
```

Comment:
@heliubj18 how about put this line in bold to emphasize it.  Here's[ how you bold text in markdown documents](https://www.markdownguide.org/basic-syntax/#:~:text=To%20bold%20text%2C%20add%20two,without%20spaces%20around%20the%20letters.&text=I%20just%20love%20**bold%20text**.).

Reply:
It is in the code block, the normal bold method like ** or __ does not work.